tests/performance/parallel_algorithms/local/benchmark_partition_copy.cpp

```diff
@@ -109,7 +109,7 @@ void run_benchmark(std::size_t vector_size, int test_count, IteratorTag)
 
     std::cout << "* Running Benchmark..." << std::endl;
 
-    int rand_base = std::rand();
+    int rand_base = std::random_device{}();
```

Comment:
One call to `std::random_device` should be sufficient. As said, its use is potentially very expensive.